python/src/ignition/gazebo/Exceptions.hh

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (C) 2021 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+#ifndef IGNITION_GAZEBO_PYTHON__EXCEPTIONS_HH_
+#define IGNITION_GAZEBO_PYTHON__EXCEPTIONS_HH_
+
+#include <stdexcept>
+#include <string>
+
+namespace ignition
+{
+namespace gazebo
+{
+namespace python
+{
+class InvalidHandle : public std::runtime_error
```

Comment:
Can't we use `std::runtime_error` directly?

Reply:
Ping @ahcorde , I think we can actually get rid of this class now, it's not used anymore